src/manual/peer2peer/extension/content-script.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
Missing copyright (I guess this was moved from webrtc/samples or another repository that uses exactly the same copyright we do).